app/renderer/components/preferences/payment/advancedSettings.js

```diff
@@ -113,15 +113,27 @@ class AdvancedSettingsFooter extends ImmutableComponent {
     }
   }
 
+  deleteWalletDuringReconcile () {
```

Comment:
Where is this being called? Seems like it should tied in to `deleteWallet`

Reply:
It's a placeholder. Waiting for @bradleyrichter or someone from UI to say the text in place of button is ok. If it's not then I'll replace the button and this would be called instead.

Reply:
Got it 👍